src/Symfony/Component/Workflow/Tests/Dumper/PlantUmlDumperTest.php

```diff
@@ -63,6 +63,23 @@ public function provideStateMachineDefinitionWithoutMarking()
         yield [$this->createComplexStateMachineDefinition(), $marking, 'complex-state-machine-marking', 'SimpleDiagram'];
     }
 
+    /**
+     * @dataProvider provideWorkflowDefinitionWithSpacesInTheStateNamesAndDescription
+     */
+    public function testDumpWorkflowWithSpacesInTheStateNamesAndDescription($definition, $expectedFileName, $title)
+    {
+        $dumper = new PlantUmlDumper(PlantUmlDumper::WORKFLOW_TRANSITION);
+        $dump = $dumper->dump($definition, null, ['title' => $title]);
+        $dump = str_replace(\PHP_EOL, "\n", $dump.\PHP_EOL);
+        $file = $this->getFixturePath($expectedFileName, PlantUmlDumper::WORKFLOW_TRANSITION);
+        $this->assertStringEqualsFile($file, $dump);
+    }
+
+    public function provideWorkflowDefinitionWithSpacesInTheStateNamesAndDescription()
+    {
+        yield [$this->createStateMachineDefinitionWithSpacesInNamesAndDescription(), 'simple-workflow-with-spaces', 'SimpleDiagram'];
+    }
```

Comment:
IMHO, we dont need a data provider (yet) since there is only one "data".

Reply:
I was thinking about that, wanted to keep consistency. But since it's just the case for PUML dump of course it doesn't make sense.

Thank you for the comment.
Updated.